chapters/equations.tex

```diff
@@ -124,8 +124,8 @@ \subsubsection{Explicit Iteration Ranges of For-Equations}\label{explicit-iterat
   constant Integer j = 4;
   Real x[j]
 equation
-  for j in 1:j loop  // The loop-variable j takes the values 1,2,3,4
-    x[j] = j;        // Uses the loop-variable j
+  for j in 1 : j loop  // j takes the values 1, 2, 3, 4
```

Comment:
```suggestion
  for j in 1:j loop    // j takes the values 1, 2, 3, 4
```
I don't think the white-space around ":" actually helps readability, and many libraries avoid it.

Reply:
Applying suggestion just so that we can move on with this.